.gitlab-ci.yml

```diff
@@ -42,6 +42,21 @@ Test EL8:
         RUNNER:
           - aws/rhel-8-x86_64
 
+Test Edge 8.4:
+  stage: test
+  extends: .terraform
+  variables:
+    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
```

Comment:
Unrelated but EXTRA_REPO_PATH_SEGMENT was removed in osbuild-composer so I suggest removing it here as well.

Reply:
Thanks, removed.